src/components/lib/lwip/cos_adapter/lwipopts.h

```diff
@@ -0,0 +1,104 @@
+#ifndef LWIPOPTS_H
+#define LWIPOPTS_H
+
+#define MEM_LIBC_MALLOC 1
+#define MEM_USE_POOLS 0
+
+#define NO_SYS 1
+#define MEM_ALIGNMENT 4
+#define MEM_SIZE 0 //64000
+/* #define MEMP_OVERFLOW_CHECK 1 */
+/* #define MEMP_SANITY_CHECK 1 */
+/* #define MEMP_NUM_PBUF (4096*4) */
+#define MEMP_NUM_UDP_PCB 512
+
+#define MEMP_NUM_TCP_PCB 512 	/* need a fair amount of these due to timed wait on close  */
+#define MEMP_NUM_TCP_PCB_LISTEN 128
+#define LWIP_ARP 0
```

Comment:
This might be useful on the actual hardware. @WenyuanShao will have to figure out if we need this there.